examples/web/nextjs-chat/src/utils/momento-web.ts

```diff
@@ -5,8 +5,9 @@ import {
   TopicClient,
   type TopicItem,
   TopicPublish,
-  TopicSubscribe,
+  type TopicSubscribe,
 } from "@gomomento/sdk-web";
+import { TopicPublishResponse, TopicSubscribeResponse } from "@gomomento/sdk";
```

Comment:
This needs to be imported from web-sdk as well: https://github.com/momentohq/client-sdk-javascript/pull/1424/files